packages/java/src/browser/java-commands.ts

```diff
@@ -75,7 +81,7 @@ export class JavaCommandContribution implements CommandContribution, MenuContrib
         });
         commands.registerCommand(JAVA_ORGANIZE_IMPORTS, {
             execute: async (changes: WorkspaceEdit) => {
-                const editor = this.editorService.activeEditor;
+                const editor = this.editorContext.widget;
```

Comment:
the keybinding context should not be reused for enabled/disabling commands, because it is too narrow.